### ISSUE TITLE
bookmarks

### ISSUE THREAD

**Issue by alltheseas** [damus-io/damus#2267]
- [ ] #427
- [ ] #1024 
- [ ] #2264

**Comment by danieldaquino** [comment]
> * [ ]  [Old bookmark links (URL) are broken #1499](https://github.com/damus-io/damus/issues/1499) @danieldaquino I think you pushed a fix for something related - preventing this issue from happening going forward?

@alltheseas yes, I remember pushing a fix that would solve that. That might be a duplicate, unless the behavior regressed or  something else going on (e.g. edge case)
